packages/react-magma-dom/src/components/TreeView/TreeItem.tsx

```diff
@@ -310,6 +310,14 @@ export const TreeItem = React.forwardRef<HTMLLIElement, TreeItemProps>(
           selectableType={selectable}
           selected={selectedItem}
           theme={theme}
+          tabIndex={itemToFocus === itemId ? 0 : -1}
```

Comment:
The challenge with moving this to here is that it causes the focus to be on the entire parent+children elements:
![image](https://github.com/user-attachments/assets/817ec0ad-9cce-4887-aba2-e73da7c72267)

We want to keep the focus on just the item:
![image](https://github.com/user-attachments/assets/57d1e551-9941-4f0e-85a6-6add84e018d5)

Reply:
done